onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -51,6 +52,7 @@ std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(TransformerLevel
       rules.push_back(onnxruntime::make_unique<ConvAddFusion>());
       rules.push_back(onnxruntime::make_unique<ConvMulFusion>());
       rules.push_back(onnxruntime::make_unique<ConvBNFusion>());
+      rules.push_back(onnxruntime::make_unique<ExpandElimination>());
```

Comment:
>ExpandElimination [](start = 47, length = 17)

we should also add this to orttraining/core/optimzier/graph_transformer_utils.cc